telegram-bot/meetup.go

```diff
@@ -62,13 +66,34 @@ func addmeetup(ID int64, msgtext string, client mongo.Client) {
 	collection := client.Database("test").Collection("meetups")
 
 	args := strings.Fields(msgtext)
-	if len(args) == 4 {
+	if len(args) == 8 {
+		//converting the arguments into integers to pass them to time.Date
+		dd, err := strconv.Atoi(args[2])
```

Comment:
Nit : Use understandeable variable names. Eg: `day`, `minutes` . Helps to maintain the codebase.

Reply:
Will keep that in mind from the next time. Thanks  :+1:.
```suggestion
		date, err := strconv.Atoi(args[2])
```